pkg/app/server/grpcapi/api.go

```diff
@@ -296,9 +296,10 @@ func (a *API) ListApplications(ctx context.Context, req *apiservice.ListApplicat
 		Cursor:  req.Cursor,
 	}
 
-	apps, cursor, err := listApplications(ctx, a.applicationStore, opts, a.logger)
+	apps, cursor, err := a.applicationStore.List(ctx, opts)
 	if err != nil {
-		return nil, err
+		a.logger.Error("failed to list applications", zap.Error(err))
```

Comment:
nit: We don't need to log this since that kind of log has already been logged at the datastore level.